anaconda.spec.in

```diff
@@ -313,6 +315,8 @@ Requires: NetworkManager-wifi
 Requires: blivet-gui-runtime >= %{blivetguiver}
 %endif
 Requires: system-logos
+# Auto-install for non-live installations (live installations use WebUI instead)
+Supplements: (anaconda unless anaconda-live)
```

Comment:
Does this need to be an rpm macro/conditional? I haven't played with weak dependencies but I thought the value was just a package name like `Requires` and `BuildRequires`